chrome/manifest.json

```diff
@@ -20,9 +20,9 @@
   },
   "permissions": [
     "activeTab",
+    "cookies",
     "declarativeNetRequestWithHostAccess",
     "webRequest",
-    "cookies",
     "storage"
   ],
```

Comment:
Just making it alphabetical (I did it when I was experimenting with moving `activeTab` to `optional_permissions` as part of the discussion in #19 ).